doc/articles/Uno-UI-Performance.md

```diff
@@ -53,6 +54,7 @@ Here's what to look for:
 	- Prefer reusing paths, duplication costs Main and GPU memory.
 	- Prefer using custom fonts over paths where possible. Fonts are rendered extremely fast and have a very low initialization time.
 - Bindings
+	- Using `x:Bind` is generally faster as it involves less or no reflection
 	- Prefer bindings with short paths.
 	- To shorten paths, use the `DataContext` property on containers, such as `StackPanel` or `Grid`.
 	- As of Uno 3.9, adding a control to loaded `Panel` or `ContentControl` does propagate the parent's DataContext immediately. If the new control has its `DataContext` immediately overridden to something else, ensure to set the DataContext before adding the control to its parent.T his will avoid having bindings be refreshed twice needlessly.
```

Comment:
```suggestion
	- As of Uno 3.9, adding a control to loaded `Panel` or `ContentControl` does propagate the parent's DataContext immediately. If the new control has its `DataContext` immediately overridden to something else, ensure to set the DataContext before adding the control to its parent. This will avoid having bindings be refreshed twice needlessly.
```